### PR TITLE
Added torchvision to hf models and updated torch version for MMD

### PR DESCRIPTION
**Pull request by nvijayrania**
- Added torchvision in HF pip requirements since it is failing latest ME component.
- Updated conda.yaml for MMD models to bring them on parity with MMD FTed model.

### CODE REVIEW

File: assets/training/model_management/src/azureml/model/mgmt/processors/pyfunc/vision/requirements.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
-mlflow==2.3.1
+mlflow==2.3.2
```
Comment (rjaincc) [inline assets/training/model_management/src/azureml/model/mgmt/processors/pyfunc/vision/requirements.txt:1]: Should we up the version to 2.6?

File: assets/training/model_management/src/azureml/model/mgmt/processors/transformers/convertors.py [hunk 2]
```diff
@@ -305,6 +306,12 @@ def save_as_mlflow(self):
         config_load_args = self._hf_conf.get(HF_CONF.HF_CONFIG_ARGS.value, {})
         config = self._hf_config_cls.from_pretrained(self._model_dir, local_files_only=True, **config_load_args)
         hf_conf[HF_CONF.TRAIN_LABEL_LIST.value] = list(config.id2label.values())
+        extra_pip_requirements = ["torchvision"]
```
Comment (MadhuM02) [inline assets/training/model_management/src/azureml/model/mgmt/processors/transformers/convertors.py:309]: Do we have torchvision in converter environment???

Comment (nvijayrania) [reply assets/training/model_management/src/azureml/model/mgmt/processors/transformers/convertors.py:309]: Yes, we do. 

Although the base image is still outdated and uses torch1.13.1 and torchvision 0.14.0
https://github.com/Azure/azureml-assets/blob/main/assets/training/model_management/environments/model-management/context/Dockerfile

File: assets/training/model_management/src/azureml/model/mgmt/processors/pyfunc/vision/requirements.txt (hunk 1 above)
Comment (nvijayrania) [reply assets/training/model_management/src/azureml/model/mgmt/processors/pyfunc/vision/requirements.txt]: Updated.

**Review by rjaincc — APPROVED** [review]

**Pull request merged**